CONTRIBUTING.md

```diff
@@ -0,0 +1,95 @@
+# Introduction
+
+### Welcome dear contributor!
+
+First off, thank you for considering contributing to xcodeproj. It's people like you that make xcodeproj such a great tool.
+
+### Why reading the guidelines is important
+
+Following these guidelines helps to communicate that you respect the time of the developers managing and developing this open source project. In return, they should reciprocate that respect in addressing your issue, assessing changes, and helping you finalize your pull requests.
+
+### A good contribution
+
+xcodeproj is an open source project and we love to receive contributions from our community — you! There are many ways to contribute, from writing tutorials or blog posts, improving the documentation, submitting bug reports and feature requests or writing code which can be incorporated into xcodeproj itself.
+
+### A bad contribution
+
+Please, don't use the issue tracker for support questions. For that purpose use Stack Overflow tagging your threads with `#xcodeproj`.
```

Comment:
While pushing people to Stack Overflow is good for a large open source project which is fairly stable, for now I think it's worth resolving support here to keep it centralised, and so we're on top of issues.
If that becomes too large then you can point people to Stack Overflow

Reply:
Good idea, let's keep it in the repo for now. I'll update the document accordingly.